### DISCUSSION TITLE
[React] vite로 생성한 프로젝트와 github 레포지토리 디렉토리 구조에 관한 질문

### DISCUSSION THREAD

**Discussion by ytzka14** [wafflestudio/seminar-2023#171] [Q&A] [answered]
이번 HW1에서, github 상에서 레포지토리를 먼저 만든 다음 git clone하고 local에서 branch를 만든 뒤 해당 레포지토리 아래에 vite로 프로젝트를 생성하는 식으로 작업했습니다. 그런데 PR 테스트를 위해서는 프로젝트가 루트 디렉토리에 존재해야 하는 조건 때문에, 이번 HW에서는 vite로 생성한 프로젝트  디렉토리 하부의 파일을 수동으로 밖으로 꺼내고 원본 디렉토리를 제거하는 식으로 수정하였습니다.
이런 조건을 맞추기 위해서 처음부터 vite로 프로젝트를 생성할 때 프로젝트 디렉토리를 수동으로 지정하거나, 혹은 vite로 생성한 폴더 자체를 github 레포지토리로 올려서 사용하는 방법이 있을지 질문드립니다.

**Comment by woohm402** [comment, marked as answer]
`create-vite` 는 타겟 디렉토리가 이미 존재해도 이용할 수 있습니다. 레포 루트의 부모 경로에서 레포 이름과 동일한 이름으로 vite 프로젝트를 생성하고, 덮어씌우기를 선택하면 자동으로 처리됩니다.

https://github.com/wafflestudio/seminar-2023/assets/39977696/11d9e55e-5d09-41d0-8de4-71fe21cd125a

그런데 과제라 그렇지, 실제로 개발할 때는 이렇게 "이미 있는 레포의 루트 경로에 `create-vite` 를 통해 프로젝트를 추가" 할 일이 없긴 합니다!
